src/interactions/core/Settings.js

```
@@ -528,7 +522,7 @@ export default class Settings extends Interaction {
         const pages = await this.#gather(ctx, channel, thread);
         return Collectors.paged(interaction, pages, ctx);
       case 'diag':
-        const embed = new MessageEmbed();
+        const embed = new EmbedBuilder();
         embed.setTitle(`Diagnostics for Shard ${interaction.guild.shardId + 1}/${interaction.client.ws.shards.size}`);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)